### PR TITLE
refs #103 Android picker & #90 Picker props

### PR DESCRIPTION
**Pull request by alvaromb**

### CODE REVIEW

File: lib/templates/bootstrap/select.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 var React = require('react-native');
-var { View, Text, PickerIOS } = React;
+var { React } = require('../../util');
```
Comment (gcanti) [inline lib/templates/bootstrap/select.js]: This import is duplicated and no more valid

**Comment by gcanti** [comment]
We should update the readme now I guess

```
```

**Comment by alvaromb** [comment]
Just rebased the stuff. Yes, ``README.md`` needs to be updated.

**Comment by alvaromb** [comment]
RN version bumped!

**Comment by gcanti** [comment]
Great, thanks! Going to merge

**Pull request merged**